src/source.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+export default class Source extends React.Component {
+
+  render() {
```

Comment:
@z0d14c I'm kinda interested why you've moved it to the top actually, since the common best practice is to put it at the bottom. :)

Also, Are you the Zodiac Killer?